File: image-train-filters-app/src/main/java/org/jesperancinha/itf/android/EmailFragment.java
package org.jesperancinha.itf.android;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.EditText;

import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

public class EmailFragment extends Fragment {

    private View mainView;

    private EditText textTo;
    private EditText textCC;
    private EditText textBCC;
    private EditText textEmail;

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        if (mainView == null) {
            mainView = inflater.inflate(R.layout.content_email, container, false);
            textTo = mainView.findViewById(R.id.editEmailTo);
            textCC = mainView.findViewById(R.id.editEmailCC);
            textBCC = mainView.findViewById(R.id.editEmailBCC);
            textEmail = mainView.findViewById(R.id.editEmailText);
        }
        return mainView;
    }

    public EditText getTextTo() {
        return textTo;
    }

    public EditText getTextCC() {
        return textCC;
    }

    public EditText getTextBCC() {
        return textBCC;
    }

    public EditText getTextEmail() {
        return textEmail;
    }
}

File: app-g2/app/src/main/java/org/jesperancinha/itf/app/EmailFragment.kt
package org.jesperancinha.itf.app

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.EditText
import androidx.fragment.app.Fragment

class EmailFragment : Fragment() {
    private var mainView: View? = null
    var textTo: EditText? = null
        private set
    var textCC: EditText? = null
        private set
    var textBCC: EditText? = null
        private set
    var textEmail: EditText? = null
        private set

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        if (mainView == null) {
            mainView = inflater.inflate(R.layout.content_email, container, false)
            textTo = requireNotNull(mainView).findViewById(R.id.editEmailTo)
            textCC = requireNotNull(mainView).findViewById(R.id.editEmailCC)
            textBCC = requireNotNull(mainView).findViewById(R.id.editEmailBCC)
            textEmail = requireNotNull(mainView).findViewById(R.id.editEmailText)
        }
        return mainView
    }
}